schema/dereferenced-release-schema.json

```
@@ -1228,15 +1228,15 @@
             },
             "description": {
               "title": "Budget Source",
-              "description": "A short free-text description of the budget to be used in a future contracting process. This may be used to provide human-readable information on: the budget category for the future contracts and/or the nature and source of the budget allocation (e.g. conditional, confirmed, or any official authorizations given to the budget allocation).",
+              "description": "A short free-text description of the budget that will fund the eventual contract(s). This may be used to provide human-readable information on: the budget category for the future contract(s) and/or the nature and source of the budget allocation (e.g. conditional, confirmed, or any official authorizations given to the budget allocation).",
               "type": [
                 "string",
                 "null"
               ]
             },
             "amount": {
               "title": "Amount",
-              "description": "The value reserved in the budget to be used in a future contracting process. A negative value indicates anticipated income to the budget as a result of this contracting process, rather than expenditure. Where the budget is drawn from multiple sources or extends over multiple years, the budget breakdown extension can be used. This field should not be used to report the total value of the budget line funding this contracting process.",
+              "description": "The value reserved in the budget that will fund the eventual contract(s). A negative value indicates anticipated income to the budget as a result of the eventual contract(s), rather than expenditure. Where the budget is drawn from multiple sources or extends over multiple years, the budget breakdown extension can be used. This field should not be used to report the total value of the budget line providing the funds for the eventual contract(s).",
```

For more clarity, I want do use the phrasing "reserved for the future contract(s)".

If I take the form "The value reserved in the budget for the future contract(s)", it could be read as "The value reserved [in the budget for the future contract(s)]", which leaves the purpose of the reservation unclear. So, I went with "The value reserved for the future contract(s), within the budget".

```suggestion
              "description": "The value reserved for the future contract(s), within the budget. A negative value indicates income to the budget as a result of the future contract(s), rather than expenditure. If the budget is drawn from multiple sources or extends over multiple years, the budget breakdown extension can be used. This field should not be used to report the total value of the budget line that will fund the future contract(s).",
```